<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JISideBarWidget</class>
 <widget class="QWidget" name="JISideBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel, QCheckBox {
color: black;
}

QComboBox, QPlainTextEdit {
color: black;
background-color: white;
padding: 4px;
}

QAbstractItemView {
color: black;
background-color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_rotation">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../rsrc/images.qrc">:/rotate-icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sl_rotation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_rotation">
       <property name="maximum">
        <number>359</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btn_rotateLeft">
       <property name="text">
        <string>Rotate Left</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/rotate-left-icon.png</normaloff>:/rotate-left-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_rotateRight">
       <property name="text">
        <string>Rotate Right</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/rotate-right-icon.png</normaloff>:/rotate-right-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btn_mirror">
       <property name="text">
        <string>Mirror</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/flip-icon.png</normaloff>:/flip-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chb_transparent">
     <property name="text">
      <string>Transparent Background</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_font">
       <property name="text">
        <string>Font:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="fcb_font">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="btn_bold">
       <property name="text">
        <string>Bold</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/bold-icon.png</normaloff>:/bold-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_italic">
       <property name="text">
        <string>Italic</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/italic-icon.png</normaloff>:/italic-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_underline">
       <property name="text">
        <string>Underline</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/underline-icon.png</normaloff>:/underline-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_hollow">
       <property name="text">
        <string>Hollow</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/hollow-icon.png</normaloff>:/hollow-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="btn_left">
       <property name="text">
        <string>Left</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/align-left-icon.png</normaloff>:/align-left-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_center">
       <property name="text">
        <string>Center</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/align-center-icon.png</normaloff>:/align-center-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_right">
       <property name="text">
        <string>Right</string>
       </property>
       <property name="icon">
        <iconset resource="../rsrc/images.qrc">
         <normaloff>:/align-right-icon.png</normaloff>:/align-right-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbl_text">
     <property name="text">
      <string>Text:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pt_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>239</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rsrc/images.qrc"/>
 </resources>
 <connections/>
</ui>
